src/Types/AbortIfFunctionTypeSpecifyingExtension.php

```diff
@@ -41,6 +41,10 @@ public function specifyTypes(
         Scope $scope,
         TypeSpecifierContext $context
     ): SpecifiedTypes {
+        if (count($node->args) < 2) {
+            return new SpecifiedTypes();
```

Comment:
What will happen here? Thanks for all your work so far.

Reply:
Could it be the handling of invalid call to `abort_if()`?

Reply:
This is to prevent crashing this extension.

This will return empty `SpecifiedTypes` But later `phpstan` will give the user an error like `Expected 2 arguments 0 given`

Reply:
For [this](https://github.com/nunomaduro/larastan/pull/330#discussion_r335929487) comment

Reply:
Confirmed, the number of arguments has to be at least 2: ```function abort_if($boolean, $code, $message = '', array $headers = [])```.